src/commands/account/create_account/fund_myself_create_account/sign_as/mod.rs

```diff
@@ -101,7 +101,7 @@ impl From<SignerAccountIdContext> for crate::commands::ActionContext {
                                             args,
                                             gas: crate::common::NearGas::from_str("30 TeraGas")
                                                 .unwrap()
-                                                .inner,
+                                                .as_gas(),
```

Comment:
In order to skip unnecessary parsing and `unwrap`s, replace such places with non-failing construction:

```rust
NearGas::from_tgas(30)
```